src/main/kotlin/io/easybreezy/project/infrastructure/ProjectRepository.kt

```diff
@@ -15,4 +21,15 @@ class ProjectRepository : Project.Repository(), Repository {
     override fun hasMembers(withRoleId: UUID): Boolean {
         return Members.select { Members.role eq withRoleId }.count() > 0
     }
+
+    override fun hasIssues(withCategoryId: UUID): Boolean {
```

Comment:
I think it's aggregation root crossing. If you use issue as aggregation root maybe will be better to move this query from project to issue. Because usually repository it's like a collection of aggregate roots. Or you can move this query to QO